source/backend/engine/CarlaEngine.cpp

```diff
@@ -596,6 +596,7 @@ bool CarlaEngine::addPlugin(const BinaryType btype,
                            && ptype != PLUGIN_GIG
                            && ptype != PLUGIN_SF2
                            && ptype != PLUGIN_SFZ
+                           && ptype != PLUGIN_JSFX
```

Comment:
any special reason for putting this line and making jsfx not able to be bridged?
Since they are external files with arbitrary code, I think allowing them as bridge is generally desired.

Reply:
Sure, bridging may have some benefits. Running jsfx compiled code is safe against crashing, but is not safe against things like infinite loops.
A thing of note is also that jsfx can't separate DSP and UI, and I don't understand too much yet how this connects with Carla's bridge concepts overall.